css/_mixin.scss

```diff
@@ -4,6 +4,16 @@
   align-items: center;
   justify-content: center;
 }
+@mixin flexcenter-row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+@mixin flexcenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
You can make just one mixin reusable for both cases.

@mixin use-flex($direction, $justify, $align) {
  display: flex;
  justify-content: $horizontal;
  align-items: $vertical;
}

and then you do @include use-flex(row, center, center);